OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+TypographyMultipleSemanticTokens.swift

```diff
@@ -0,0 +1,86 @@
+//
+// Software Name: OUDS iOS
+// SPDX-FileCopyrightText: Copyright (c) Orange SA
+// SPDX-License-Identifier: MIT
+// 
+// This software is distributed under the MIT license,
+// the text of which is available at https://opensource.org/license/MIT/
+// or see the "LICENSE" file for more details.
+// 
+// Authors: See CONTRIBUTORS.txt
+// Software description: A SwiftUI components library with code examples for Orange Unified Design System 
+//
+
+import Foundation
+import OUDSTokensRaw
+import OUDSTokensSemantic
+
+// [File not generated by the tokenator]
```

Comment:
is it right ? gnerated ?

Reply:
Ce fichier n'est pas généré par le _tokenator_ car il ne peut pas générer de tels objets dépendant de _size class_ ou _color scheme_ dans le mesure où quand bien même ces tokens sont définis sur Figma par l'équipe de design cet outil n'est _a priori_ pas capable de les gérer. De fait le JSON produit pas Figma ne peut pas ressortir ces objets, donc le _tokenator_ ne peut rien produire. D'où _in fine_ le fait qu el'on expose ces objets en pointant nous-mêmes vers les tokens pertinents.